core/src/task.cpp

```diff
@@ -281,6 +281,12 @@ moveit::core::MoveItErrorCode Task::execute(const SolutionBase& s) {
 	moveit_task_constructor_msgs::action::ExecuteTaskSolution::Goal goal;
 	s.toMsg(goal.solution, pimpl()->introspection_.get());
 
+	// Reset joint values in scene diff
```

Comment:
Not required but I prefer to tell other developers why we need to reset the values.